### PR TITLE
making the codebase work on Windows

### PR DESCRIPTION
**Pull request by jannalulu**
webpage renders but backend still doesn't work. Also still creating new migration files in common/src instead of using existing files in common/src/db

### CODE REVIEW

File: common/src/db/drizzle.config.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ import path from 'path'
 
 export default defineConfig({
   dialect: 'postgresql',
-  schema: path.join('src', 'db', 'schema.ts'),
-  out: path.join('src', 'db', 'migrations'),
+  schema: path.join(__dirname, 'schema.ts').replace(/\\/g, '/'),
+  out: './src/migrations',
```
Comment (jahooma) [inline common/src/db/drizzle.config.ts]: This line looks suspicious

File: common/src/db/env.mjs [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+console.log('Loading stack.env from:', stackEnvPath)
+const stackResult = dotenv.config({ path: stackEnvPath })
+console.log('Stack env loaded:', stackResult)
+
+// Force set the environment from stack.env
+if (stackResult.parsed?.ENVIRONMENT) {
+  process.env.ENVIRONMENT = stackResult.parsed.ENVIRONMENT
+}
+
+console.log('ENVIRONMENT after loading stack.env:', process.env.ENVIRONMENT)
```
Comment (jahooma) [inline common/src/db/env.mjs]: Remove console.logs before merging

Comment (jannalulu) [reply common/src/db/env.mjs]: removed

File: common/src/db/drizzle.config.ts (hunk 1 above)
Comment (brandonkachen) [reply common/src/db/drizzle.config.ts]: @jannalulu can you try 'src/db/migrations' here instead?

Comment (jannalulu) [reply common/src/db/drizzle.config.ts]: done

**Pull request merged**